src/google-mobility.html

```diff
@@ -71,15 +70,46 @@ <h4 class="mdc-typography--headline4 text-blue">Changes by location</h4>
       <p class="mdc-typography--caption">*Try dragging the chart to zoom in. Reset by using the right-click.</p>
 
       <div class="info-container">
-        <h4 class="mdc-typography--headline3 text-center">What is a Mobility Report?</h4>
+        <h4 class="mdc-typography--headline3 text-center">What are Google COVID-19 Mobility Reports?</h4>
         <p class="mat-typography--body1 text-center">
-          The Community Mobility Reports reflect the changes in response to policies aimed at combating COVID-19. The reports chart
-          movements over time by geography, and across different places such as retail and recreation, groceries and pharmacies, parks,
-          transit stations, workplaces, and residential.
+          The Google COVID-19 <a target="_blank" href="https://www.google.com/covid19/mobility">Community Mobility Reports</a> aim to
```

Comment:
Can you make the links in this section white?
<img width="773" alt="Screen Shot 2020-07-18 at 7 16 50 PM" src="https://user-images.githubusercontent.com/11969737/87863677-6c8a4880-c92b-11ea-91a7-e9889d4fb008.png">